libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm

```diff
@@ -0,0 +1,8 @@
+BATT_VOLT_PIN 12
+BATT_CURR_PIN 13
+
+CAN_P1_DRIVER 1
+CAN_P2_DRIVER 1
+
+BATT_MONITOR 8
+GPS1_TYPE 9
```

Comment:
You should not be setting the GPS type - that is up to the user

Reply:
they supply a dronecan gps with the autopilot....should remain

Reply:
We primarily consider our users, aiming to reduce the difficulty in using our products. Our PMU and GPS communicate via dronecan, they can communicate with each other without requiring additional parameter configuration, so we have set default parameters: CAN P1 DRIVER=1, GPS1_TYPE=9.

Reply:
if the CAN PMU is also provided with autopilot , it should be setup also, not an analog PMU

Reply:
Yes , we alsp supply a dronecan PMU .